tdutils/td/utils/port/Clocks.h

```diff
@@ -26,6 +27,62 @@ struct Clocks {
   static double system();
 
   static int tz_offset();
+
+#if defined(__i386__)
```

Comment:
all of these defines are not available under windows and MSVC. On windows you can use __rdtsc and __forceinline (__inline__ also seems missing)

Reply:
Hi